lighthouse-core/lib/tracehouse/trace-processor.js

```diff
@@ -530,6 +571,8 @@ class TraceProcessor {
     // Ensure our traceEnd reflects all page activity.
     const traceEnd = this.computeTraceEnd(trace.traceEvents, timeOriginEvt);
 
+    /** @param {number=} ts */
```

Comment:
```suggestion
    /** @param {number|undefined} ts */
```

I think connor just went through all of these `=` and updated them for us :)